buildkite/bazelci.py

```diff
@@ -130,6 +130,11 @@
         "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/cloud-robotics-postsubmit.yml",
         "pipeline_slug": "cloud-robotics-core",
     },
+    "Envoy": {
+        "git_repository": "https://github.com/envoyproxy/envoy.git",
+        "http_config": "https://github.com/envoyproxy/envoy/blob/master/.bazelci/presubmit.yml",
```

Comment:
I think you need raw content URL here? like: https://raw.githubusercontent.com/envoyproxy/envoy/master/.bazelci/presubmit.yml

Reply:
Yep, my bad, fixed!